desc/objectives/_generic.py

```diff
@@ -272,5 +270,5 @@ def compute(self, R_lmn, Z_lmn, L_lmn, i_l, c_l, Psi, **kwargs):
             self._iota,
             self._current,
         )
-        I = 2 * jnp.pi / mu_0 * data["I"]
+        I = compress(self.grid, data["current"], surface_label="rho")
         return self._shift_scale(I)
```

Comment:
Don't need to wait for another CI for this, but I'd suggest eventually changing `I` to `current` for consistency.